timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -553,15 +555,10 @@ def _postprocess_predictions(self, predictions: np.ndarray, repeated_item_ids: p
             predictions["mean"] = predictions["0.5"]
         else:
             predictions = pd.DataFrame(predictions, columns=["mean"])
-            predictions = self._add_gaussian_quantiles(predictions, repeated_item_ids=repeated_item_ids)
```

Comment:
Why did this part change with the scaling?

Reply:
Previously, we had a rather convoluted logic that can be summarized as

- in fit(): original data -> scaled data -> compute residuals -> save "scaled" residuals
- in predict(): original data -> scaled data -> scaled predictions -> add scaled residuals -> undo scaling

We now simplified this to
- in fit(): original data -> scaled data -> compute residuals -> undo scaling on residuals -> save residuals that are on the same scaled as original data
- in predict(): original data -> scaled data -> scaled predictions -> undo scaling -> add residuals

The latter approach is slightly cleaner and does not requires us to mess with the internals of the scaler.

Reply:
Thank you for clarifying!